exp-android-sdk/src/main/java/com/scala/exp/android/sdk/MainActivity.java

```diff
@@ -48,46 +52,86 @@ protected void onCreate(Bundle savedInstanceState) {
+        Log.i(LOG_TAG, "START EXP SDK");
+        Exp.start(startOptions)
                 .subscribe(new Action1<Boolean>() {
                     @Override
                     public void call(Boolean o) {
 
-                       Subscriber currentExperienceSubs = new Subscriber<JSONObject>() {
-                           @Override
-                           public void onCompleted() {}
-                           @Override
-                           public void onError(Throwable e) {Log.d(LOG_TAG, e.toString());}
-                           @Override
-                           public void onNext(JSONObject o) {
-                               try {
-                                   Object target = o.get("target");
-                               } catch (JSONException e) {
-                                   e.printStackTrace();
-                               }
-                               Log.d(LOG_TAG, o.toString());
-                           }
-                       };
+                        Subscriber currentExperienceSubs = new Subscriber<JSONObject>() {
+                            @Override
+                            public void onCompleted() {
+                            }
+
+                            @Override
+                            public void onError(Throwable e) {
+                                Log.d(LOG_TAG, e.toString());
+                            }
+
+                            @Override
+                            public void onNext(JSONObject o) {
+                                try {
+                                    Object target = o.get("target");
+                                } catch (JSONException e) {
+                                    e.printStackTrace();
+                                }
+                                Log.d(LOG_TAG, o.toString());
+                            }
+                        };
 
                         Subscriber socketConnection = new Subscriber() {
                             @Override
-                            public void onCompleted() {}
+                            public void onCompleted() {
+                            }
+
                             @Override
-                            public void onError(Throwable e) {}
+                            public void onError(Throwable e) {
+                            }
+
                             @Override
                             public void onNext(Object o) {
                                 Log.d(LOG_TAG, o.toString());
                             }
                         };
-                        Exp.connection(Utils.ONLINE,socketConnection);
-                        Exp.getCurrentExperience(currentExperienceSubs);
 
-                        IChannel channel = Exp.getChannel(Utils.SOCKET_CHANNELS.SYSTEM);
-                        channel.fling("1111");
+                        final IChannel channel1 = Exp.getChannel("channel1",0,1);
```

Comment:
can we expose the 0 and 1 as consts from Exp?  Something like Exp.SYSTEM or whatever the 0 and 1 mean.  

Also they are integers because there will be more options than just 0 and 1 right?